pkgs/servers/http/nginx/modules.nix

```diff
@@ -985,8 +985,12 @@ let self = {
       name = "vod";
       owner = "kaltura";
       repo = "nginx-vod-module";
-      rev = "1.31";
-      hash = "sha256-ZpeO8QWQ+fGkz08u/zFOq7vj4aHcodzSHNrc1SgGUyc=";
+      rev = "1.32";
+      hash = "sha256-ZpG0oj60D3o7/7uyE8AybCiOtncVe1Jnjaz22sIFypk=";
+      postFetch = ''
+        substituteInPlace $out/vod/media_set.h \
+          --replace "MAX_CLIPS (128)" "MAX_CLIPS (1024)"
+      '';
     };
 
     inputs = [ ffmpeg fdk_aac openssl libxml2 libiconv ];
```

Comment:
I wonder whether we can get away with using `ffmpeg-headless`. That would save more than 200 MB.

```
$ nix path-info -Sh /nix/store/lymwbqc9hmfaknr1n0wlmcrdklfy9g54-ffmpeg-6.0-bin
/nix/store/lymwbqc9hmfaknr1n0wlmcrdklfy9g54-ffmpeg-6.0-bin	621.9M
$ nix path-info -Sh /nix/store/fxg8v7zfq6fan7fnmgsixlpizalqdrsw-ffmpeg-headless-6.0-bin
/nix/store/fxg8v7zfq6fan7fnmgsixlpizalqdrsw-ffmpeg-headless-6.0-bin	408.1M
```

Reply:
It at least it compiles without errors when using ffmpeg-headless. Currently deploying the new vod module to my server to check if it works 🤞🏻

Reply:
Looks good. At least the functionality that Frigate uses from the VOD module, is not affected by switching to ffmpeg-headless :)